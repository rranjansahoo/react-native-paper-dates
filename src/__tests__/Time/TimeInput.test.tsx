import * as React from 'react'

import renderer from 'react-test-renderer'
import TimeInput from '../../Time/TimeInput'
import { clockTypes } from '../../Time/timeUtils'

it('renders TimeInput', () => {
  const tree = renderer
    .create(
      <TimeInput
        value={12}
        clockType={clockTypes.hours}
        pressed
        inputType={'picker'}
        onChanged={() => null}
      />
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
