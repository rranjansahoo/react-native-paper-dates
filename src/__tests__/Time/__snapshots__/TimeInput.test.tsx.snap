// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TimeInput 1`] = `
<View
  style={
    {
      "height": 80,
      "position": "relative",
      "width": 96,
    }
  }
>
  <TextInput
    keyboardAppearance="default"
    keyboardType="number-pad"
    maxFontSizeMultiplier={1.5}
    maxLength={2}
    onBlur={[Function]}
    onChangeText={[Function]}
    onFocus={[Function]}
    style={
      [
        {
          "textAlign": "center",
          "textAlignVertical": "center",
          "width": 96,
        },
        {
          "backgroundColor": "rgba(232, 222, 248, 1)",
          "borderColor": "rgba(33, 0, 93, 1)",
          "borderRadius": 8,
          "borderWidth": 2,
          "color": "rgba(33, 0, 93, 1)",
          "fontSize": 57,
          "height": 80,
        },
      ]
    }
    value="12"
  />
</View>
`;
